libs/pika/command_line_handling/src/parse_command_line.cpp

```diff
@@ -340,8 +340,11 @@ namespace pika::detail {
                 "no cross boundary stealing is allowed (default value: 0)")
 #if defined(PIKA_HAVE_MPI)
             ("pika:mpi-completion-mode", value<std::size_t>(),
-                "the pika MPI polling completion mode (only available if MPI "
-                "built with MPI support)")
+                "the pika MPI polling completion/handler mode "
+                "(only available if MPI built with MPI support)")
+            ("pika:mpi-enable-pool",
+                "enable the creation of a dedicated pool for mpi-polling "
+                "(only available if MPI built with MPI support and available threads>1).")
```

Comment:
Regarding "only available if ... available threads>1": will using this flag result in error or not?

Reply:
it prints a warning, but I'd better switch it to use PIKA_LOG as you suggested for other messages - we can throw instead ...